src/ray/gcs/gcs_server/gcs_node_manager.h

```diff
@@ -39,9 +41,11 @@ class GcsNodeManager : public rpc::NodeInfoHandler {
   ///
   /// \param gcs_publisher GCS message publisher.
   /// \param gcs_table_storage GCS table external storage accessor.
-  explicit GcsNodeManager(std::shared_ptr<GcsPublisher> gcs_publisher,
-                          std::shared_ptr<gcs::GcsTableStorage> gcs_table_storage,
-                          std::shared_ptr<rpc::NodeManagerClientPool> raylet_client_pool);
+  explicit GcsNodeManager(
+      std::shared_ptr<GcsPublisher> gcs_publisher,
+      std::shared_ptr<gcs::GcsTableStorage> gcs_table_storage,
+      std::shared_ptr<rpc::NodeManagerClientPool> raylet_client_pool,
+      std::shared_ptr<ClusterTaskManager> cluster_task_manager = nullptr);
```

Comment:
Can you docstring this and mention this is configured only when GCS-based actor scheduling is enabled?

Reply:
@rkooo567 See the discussion with Yi below.